core/events/events_procedure_change_return.ts

```diff
@@ -5,19 +5,39 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import {IProcedureModel} from '../interfaces/i_procedure_model.js';
+import type {IProcedureModel} from '../interfaces/i_procedure_model.js';
 import * as registry from '../registry.js';
-import {Workspace} from '../workspace.js';
+import type {Workspace} from '../workspace.js';
 
 import {ProcedureBase} from './events_procedure_base.js';
 import * as eventUtils from './utils.js';
 
 
 export class ProcedureChangeReturn extends ProcedureBase {
```

Comment:
Should ~~this class~~ these classes not have a JSDoc explaining its purpose?  It would make it easier for the unfamiliar reviewer…

(Documentation of the properties would be helpful too.)

Reply:
Added docs! If you think there's more that could be added I'm happy to do more =)

Reply:
Yes, even these simple (and probably from your point of view slightly redundant) docs are useful to this reader.